### ISSUE TITLE
Panic When Doing Pull From Appveyor

### ISSUE THREAD

**Issue by andrejohansson** [git-lfs/git-lfs#3031]
When doing a git lfs pull from AppVeyor we get a panic error.

I guess this might have to do with git lfs trying to show some progressbar and AppVeyor doesnt have a window handle?

```
git lfs pull
panic: runtime error: index out of range
goroutine 8 [running]:
github.com/git-lfs/git-lfs/tools/humanize.FormatBytes(0xffffffffffffffff, 0xc042494398, 0x8)
	C:/Users/ttaylorr/go/src/github.com/git-lfs/git-lfs/src/github.com/git-lfs/git-lfs/tools/humanize/humanize.go:110 +0x1ec
github.com/git-lfs/git-lfs/tq.(*Meter).str(0xc04206acf0, 0xc04232db00, 0x42a666)
	C:/Users/ttaylorr/go/src/github.com/git-lfs/git-lfs/src/github.com/git-lfs/git-lfs/tq/meter.go:241 +0xf4
github.com/git-lfs/git-lfs/tq.(*Meter).update(0xc04206acf0, 0xc04220cf00)
	C:/Users/ttaylorr/go/src/github.com/git-lfs/git-lfs/src/github.com/git-lfs/git-lfs/tq/meter.go:219 +0x51
github.com/git-lfs/git-lfs/tq.(*Meter).Skip(0xc04206acf0, 0xffffffffffffffff)
	C:/Users/ttaylorr/go/src/github.com/git-lfs/git-lfs/src/github.com/git-lfs/git-lfs/tq/meter.go:127 +0x75
github.com/git-lfs/git-lfs/tq.(*TransferQueue).Skip(0xc04217fd00, 0xffffffffffffffff)
	C:/Users/ttaylorr/go/src/github.com/git-lfs/git-lfs/src/github.com/git-lfs/git-lfs/tq/transfer_queue.go:718 +0x40
github.com/git-lfs/git-lfs/tq.(*TransferQueue).enqueueAndCollectRetriesFor(0xc04217fd00, 0xc04201a380, 0x7, 0x64, 0x0, 0x0, 0x0, 0x0, 0x0)
	C:/Users/ttaylorr/go/src/github.com/git-lfs/git-lfs/src/github.com/git-lfs/git-lfs/tq/transfer_queue.go:464 +0x696
github.com/git-lfs/git-lfs/tq.(*TransferQueue).collectBatches.func1(0xc042003560, 0xc04217fd00, 0xc042002720, 0xc042036240)
	C:/Users/ttaylorr/go/src/github.com/git-lfs/git-lfs/src/github.com/git-lfs/git-lfs/tq/transfer_queue.go:354 +0x5b
created by github.com/git-lfs/git-lfs/tq.(*TransferQueue).collectBatches
	C:/Users/ttaylorr/go/src/github.com/git-lfs/git-lfs/src/github.com/git-lfs/git-lfs/tq/transfer_queue.go:360 +0x376
Command exited with code 2
```

**Comment by ttaylorr** [comment]
Hi @andrejohansson, thanks for opening this issue. I think that this is related to the bug that was fixed in #2902 and is fixed in the latest release (v2.4.1). Can you make sure that AppVeyor is running the latest version?

If that's not the case, please let me know here and I'd be more than happy to dive in and figure out what's going on.

**Comment by andrejohansson** [comment]
I can confirm that they were running 2.4.0, I´ll try to run 2.4.1 and se if it helps, thanks for your reply.
